isis/tests/PixelTests.cpp

```diff
@@ -465,7 +465,6 @@ TEST(Pixel, static_ToString) {
   EXPECT_EQ(std::string("Lis"), Pixel::ToString(Isis::Lis));
   EXPECT_EQ(std::string("Lrs"), Pixel::ToString(Isis::Lrs));
   EXPECT_EQ(std::string("Null"), Pixel::ToString(Isis::Null));
-  EXPECT_EQ(std::string("Invalid"), Pixel::ToString(-1.0e+1000));
```

Comment:
Is there any way we can test "Invalid" still without exceeding the valid range for doubles? Or is that what "Invalid" checks for?

Reply:
This is checking for values below valid minimum. The problem is, all of the double values below valid minimum are special pixels or NaN.

The only way this happens is a double overflow, which I think is such an extreme outlier it's not worth testing.

Reply:
:+1: Sounds good to me, then!